src/plugins/data/public/search/session/sessions_mgmt/components/actions/inspect_button.tsx

```diff
@@ -33,7 +33,7 @@ const InspectFlyout = ({ uiSettings, searchSession }: InspectFlyoutProps) => {
   const renderInfo = () => {
     return (
       <Fragment>
-        <CodeEditor
+        <CodeEditor // TODO theme provided by src/plugins/data/public/search/search_service.ts
```

Comment:
@elastic/kibana-data-discovery not sure which editor is that? Can you check if it is ok?